docs/application/dotnet/guides/multimedia/media-content.md

```diff
@@ -107,14 +107,14 @@ To enable your application to use the media content functionality:
         Tizen.Log.Info(LogTag, $"Folder updated: Id = {args.Id}, Operation = {args.OperationType}");
     }
 
-    MediaDatabase.MediInfoUpdated += OnMediaInfoUpdated;
+    MediaDatabase.MediaInfoUpdated += OnMediaInfoUpdated;
     MediaDatabase.FolderUpdated += OnFolderUpdated;
     ```
 
 2. When you no longer want to receive notifications, deregister the event handlers:
 
     ```
-    MediDatabase.MediInfoUpdated -= OnMediaInfoUpdated;
+    MediDatabase.MediaInfoUpdated -= OnMediaInfoUpdated;
```

Comment:
line117,118: Change MediDatabase to MediaDatabase.

Reply:
I'll fix it.